back-end/index.js

```diff
@@ -1,32 +1,46 @@
 import app from "./server.js"
-import mongodb from "mongodb"
+// import mongodb from "mongodb"
+import mongoose from "mongoose"
 import dotenv from "dotenv"
-import TranscriptsDAO from "./dao/transcriptsDAO.js"
-import TextTranscriptsDAO from "./dao/textTranscriptsDAO.js"
-import ProjectsDAO from "./dao/projectsDAO.js";
+// import TranscriptsDAO from "./dao/transcriptsDAO.js"
+// import TextTranscriptsDAO from "./dao/textTranscriptsDAO.js"
+// import ProjectsDAO from "./dao/projectsDAO.js";
 
 dotenv.config()
-const MongoClient = mongodb.MongoClient
+// const MongoClient = mongodb.MongoClient
 
-const port = process.env.PORT || 8000
+const port = process.env.PORT || 5000
 
+// format: mongoose.connect(uri, options{dbName: connects to specific database});
+mongoose.connect(process.env.MONGO_DB_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  dbName: process.env.PROJECTS_NS,
+});
```

Comment:
Maybe we should change the env variable name so its just not reflecting projects?

Reply:
yes, maybe something like ''?

> Oh also before this change is pulled into development we have to update the docker file to make sure its including these subdirectories

Yup, I'll make sure to merge development into my branch we merge this pr.

Reply:
Perfect!